src/settings.cpp

```diff
@@ -243,6 +263,24 @@ int Settings::toggle_cmd(Input argv, Output output) {
     return 0;
 }
 
+void Settings::toggle_complete(Completion& complete) {
+    // types that can be toggled
+    std::vector<Type> l = {
```

Comment:
I suggest using std::set here and then later on std::set::find() instead of std::find(). Better readability.

Reply:
I think it's overkill to construct a set (i.e. binary tree/heap) with three elements. Because of #249 / #250, I don't need the vector/set anyway because after #250, only boolean settings can be toggled.

Reply:
Just a sidenote (even if it doesn't matter anymore here): A decision like this sounds like a textbook example for when *not* to take performance/overhead into account. Creating a set is NlogN (I think), not N^9000, so for three elements it just doesn't matter. That's all we should care about when writing code like that, and I appreciate the fact that's even saying so in the Goals & Principles.